pkg/cmd/server/origin/master.go

```diff
@@ -309,11 +355,17 @@ func (c *MasterConfig) Run(protected []APIInstaller, unprotected []APIInstaller)
 
 	handler = open
 
+	handler = indexAPIPaths(handler)
```

Comment:
can we wrap these around the `handler` that gets passed to `open.Handle("/", handler)` so we don't add two levels of indirection to subpaths that have their own registered handlers?